public/index.html

```diff
@@ -8,6 +8,9 @@
     <meta name="application-name" content="DreamScholars" />
     <meta name="description" content="Scholarships for Undocomented Students" />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
```

Comment:
this is what was causing the font to change 🙃

Reply:
Oh? How so? What does this do?